services/api-koa/src/resources/token/token.service.ts

```diff
@@ -7,18 +7,22 @@ import { Token, TokenType } from './token.types';
 
 const service = db.createService<Token>(DATABASE_DOCUMENTS.TOKENS, { schema });
 
-const createToken = async (userId: string, type: TokenType) => {
+const createToken = async (userId: string, type: TokenType, isShadow?: boolean) => {
```

Comment:
I think `isShadow` should have default value `null`.
If `isShadow` value will be not passed to `authService.setTokens ` `service.insertOne` will set `{ isShadow: undefined }`.